src/screens/guides/components/custom-components/ecology.md

```diff
@@ -0,0 +1,172 @@
+# Custom Components
+
+Every element that a Victory component renders may be altered or completely replaced. Most components expose `dataComponent`, `labelComponent`, `groupComponent`, and `containerComponent` props. The primitive components that Victory components render by default are simple, stateless components with a consistent set of props whenever possible. These [primitive components] are exported for users to alter, wrap, extend and reference when creating custom components. 
```

Comment:
[primitive components] also needs a link - this may be true throughout the docs, just wanted to give you a heads up if you didn't realize!